packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -185,6 +198,7 @@ class _MaterialScrollbar extends RawScrollbar {
          isAlwaysShown: isAlwaysShown,
          thickness: thickness,
          radius: radius,
+         buttonsShape: buttonsShape,
```

Comment:
Why does buttonVisibility not also go to the super class?

Reply:
The super class's property is not a material state property.